From: Joe Lawrence <joe.lawrence@redhat.com>
Date: Thu, 31 Jan 2019 11:41:24 -0500
Subject: selftests/livepatch: add DYNAMIC_DEBUG config dependency
Git-commit: bae054372aba0100e7b320f768c4875b5f0c23b3
Patch-mainline: v5.1-rc1
References: bsc#1071995 fate#323487

The livepatch selftest scripts turn on dynamic_debug of livepatch
kernel source to determine expected behavior.  TEST_LIVEPATCH should
therefore include DYNAMIC_DEBUG in its list of dependencies.

Signed-off-by: Joe Lawrence <joe.lawrence@redhat.com>
Acked-by: Miroslav Benes <mbenes@suse.cz>
Signed-off-by: Petr Mladek <pmladek@suse.com>
---
 lib/Kconfig.debug | 1 +
 1 file changed, 1 insertion(+)

diff --git a/lib/Kconfig.debug b/lib/Kconfig.debug
index 0298eb8eaa4e..0a3af2965ba0 100644
--- a/lib/Kconfig.debug
+++ b/lib/Kconfig.debug
@@ -1994,6 +1994,7 @@ config TEST_MEMCAT_P
 config TEST_LIVEPATCH
 	tristate "Test livepatching"
 	default n
+	depends on DYNAMIC_DEBUG
 	depends on LIVEPATCH
 	depends on m
 	help

